modules/ProvMon/Http/Controllers/ProvMonController.php

```diff
@@ -769,21 +837,58 @@ public function realtime($host, $com, $ip, $cacti)
     }
 
     /**
-     * The CMTS Realtime Measurement Function
-     * Fetches all realtime values from CMTS with SNMP
+     * Fetch realtime values via GenieACS
+     *
+     * @param modem: modem object
+     * @param refresh: refresh values from device instead of unsing cached ones
```

Comment:
```suggestion
     * @param refresh: refresh values from device instead of using cached ones
```
which type are the refresh values? I assume Array?

Reply:
bool, fixed in 6758a7e